ansible/roles/add_user/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+- name: Add group
+  group: name="{{ ssh_user }}" state=present
+  become: yes
+
+- name:  Create user
+  become: yes
+  user: state=present name="{{ ssh_user }}" group="{{ ssh_user }}" groups=sudo shell=/bin/bash createhome=yes password="{{ password }}" update_password=always
```

Comment:
@G33tha  This will add the user to sudo group. Is this intended? Do we want to add users to sudo group?

Reply:
yes, we wanted add users to sudo group